dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MalformedRawRequestTests.kt

```diff
@@ -49,7 +49,7 @@ class MalformedRawRequestTests {
             .build()
 
         val response = client.newCall(request).execute()
-        val responseBodyString = response.body?.string() ?: ""
+        val responseBodyString = response.body.string()
```

Comment:
Das ist aber ein Code Duplikat das man leicht in eine Funktion packen könnte mit request und expectedString als parameter oder sowas

Reply:
Wie besprochen: Test reparieren out of focus für diesen Branch.